src/main/kotlin/no/nav/familie/ef/søknad/mapper/FeltMapperUtil.kt

```diff
@@ -25,7 +23,7 @@ fun PeriodeFelt.tilSøknadsfelt(): Søknadsfelt<MånedÅrPeriode> =
                                    this.til.tilLocalDate().month,
                                    this.til.tilLocalDate().year))
 
-fun <T> ListFelt<T>.tilSøknadsfelt(): Søknadsfelt<List<T>> = Søknadsfelt(this.label, this.verdi)
+fun <T> ListFelt<T>.tilSøknadsfelt(): Søknadsfelt<List<T>> = Søknadsfelt(this.label, this.verdi, this.alternativer)
```

Comment:
Jeg tror det er nyttig med alternativer også på tekstfelt ettersom vi har noen "radio"-button valg hvor alternativer kan være nyttig å få med.

Reply:
(Men da må alternativer være optional isåfall)

Reply:
Jepp - enig. Jeg foreslår at legger det på når vi trenger det. UI sender uansett ikke noe ennå.